RELEASE_NOTES.md

```diff
@@ -1,3 +1,9 @@
+# ChefDK 3.0.24 Release Notes
```

Comment:
This should just be 3.0 since we'll be at some unknown version number before we release

Reply:
👍 updated